### PR TITLE
RF: Fix ITK warp conversion to nitransforms format

### PR DESCRIPTION
**Pull request by effigies**
Alternative to (and builds on) #3296.

Closes #3296.

### CODE REVIEW

File: fmriprep/utils/transforms.py [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 
-import warnings
 from pathlib import Path
 
 import h5py
 import nibabel as nb
 import nitransforms as nt
 import numpy as np
 from nitransforms.io.itk import ITKCompositeH5
+from tranforms3d.affines import compose as compose_affine
```
Comment (effigies) [inline fmriprep/utils/transforms.py:10]:
```suggestion
from transforms3d.affines import compose as compose_affine
```

**Review by mgxd — APPROVED** [review]
LGTM - I have not tested it, but can appreciate the improved clarity.
The only concern I have is this code is copied in a number of places (here, nibabies, resampler package) - we should focus some energy on having a single distribution and importing from there (maybe an ohbm hackathon project?)

**Comment by effigies** [comment]
Actually just looked at the MNI-resampled boldref in https://app.circleci.com/pipelines/github/nipreps/fmriprep/2204/workflows/b0c88b0f-7a65-4904-a5d4-7dc26b8233f4/jobs/11504/artifacts, and it's degraded relative to your PR and master. Let me try to understand this...

**Comment by effigies** [comment]
Hmm. I loaded the file with my function and yours, and they seem identical. I'm going to re-run the test, in case it's just a one-off weirdness.

**Comment by effigies** [comment]
Okay, it appears to have been a one-off bad boldmask, not a problem with this function at all.

**Comment by effigies** [comment]
> The only concern I have is this code is copied in a number of places (here, nibabies, resampler package) - we should focus some energy on having a single distribution and importing from there (maybe an ohbm hackathon project?)

And yes. We should get this fixed up in nitransforms so we can stop having our own special loader.

**Pull request merged**